src/app/lib/config/services/ws.ts

```diff
@@ -334,56 +334,74 @@ export const service: DefaultServiceConfig = {
             id: 'afaanoromoo',
             href: 'https://www.bbc.com/afaanoromoo',
             label: 'Oduu Afaan Oromootiin',
+            lang: 'om',
+            latinTransliteration: 'Oduu Afaan Oromootiin',
           },
           {
             id: 'amharic',
             href: 'https://www.bbc.com/amharic',
             label: 'ዜና በአማርኛ',
+            lang: 'am',
+            latinTransliteration: 'Zena be Amharic',
           },
           {
             id: 'afrique',
             href: 'https://www.bbc.com/afrique',
             label: "L'actualité en français",
+            lang: 'fr',
           },
           {
             id: 'hausa',
             href: 'https://www.bbc.com/hausa',
             label: 'Labarai da Hausa',
+            lang: 'ha',
           },
           {
             id: 'igbo',
             href: 'https://www.bbc.com/igbo',
-            label: 'Akụkọ n’Igbo',
+            label: "Akụkọ n'Igbo",
+            lang: 'ig',
           },
           {
             id: 'gahuza',
             href: 'https://www.bbc.com/gahuza',
             label: 'Amakuru mu Kinyarwanda/ Kirundi',
+            lang: 'rw',
+            latinTransliteration: 'Amakuru mu Kinyarwanda/ Kirundi',
           },
           {
             id: 'pidgin',
             href: 'https://www.bbc.com/pidgin',
             label: 'News in Pidgin',
+            lang: 'en',
```

Comment:
Is this correct - should it be the same as lang from the service file: https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/pidgin.ts#L9
```suggestion
            lang: 'pcm',
```